### PR TITLE
[FIX] add missing processing entity for MEG physio files and render physio filename templates for fNIRS

### LINKED ISSUE #1354: [BUG] physio is handled differently for MEG between 1.7 and latest
### Describe your problem in detail.

Related
- physio is not supported for fNIRS

From version 1.7.0

https://bids-specification.readthedocs.io/en/v1.7.0/04-modality-specific-files/02-magnetoencephalography.html#meg-recording-data

```text
sub-<label>/
    [ses-<label>/]
        meg/
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>][_recording-<label>]_physio.json
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>][_recording-<label>]_physio.tsv.gz
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>][_recording-<label>]_stim.json
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_proc-<label>][_recording-<label>]_stim.tsv.gz
```

From latest

https://bids-specification.readthedocs.io/en/latest/modality-specific-files/magnetoencephalography.html#meg-recording-data
```
sub-<label>/
    [ses-<label>/]
        meg/
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_recording-<label>]_physio.json
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_recording-<label>]_physio.tsv.gz
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_recording-<label>]_stim.json
            sub-<label>[_ses-<label>]_task-<label>[_acq-<label>][_run-<index>][_recording-<label>]_stim.tsv.gz

```

### PR DESCRIPTION
**Pull request by Remi-Gau**
fixes #1354

### CODE REVIEW

File: src/schema/rules/files/raw/task.yaml [hunk 1]
```diff
@@ -29,7 +29,6 @@ timeseries:
     - beh
     - eeg
     - ieeg
-    - meg
```
Comment (sappelhoff) [inline src/schema/rules/files/raw/task.yaml:32]: why is meg being removed here?

Comment (Remi-Gau) [reply src/schema/rules/files/raw/task.yaml:32]: Because it is handled separately below to allow for the proc extension.

**Review by sappelhoff — APPROVED** [review]

**Comment by effigies** [comment]
Isn't proc like rec, where it's the same scan but processed differently? What does it mean for that to apply to physio or stim data?

**Comment by Remi-Gau** [comment]
> Isn't proc like rec, where it's the same scan but processed differently? What does it mean for that to apply to physio or stim data?

I think we had discussed this extensively here: https://github.com/bids-standard/bids-specification/pull/894

**Comment by effigies** [comment]
Okay. If it's been discussed I won't reopen it.

**Review by effigies — APPROVED** [review]

**Review by rob-luke — APPROVED** [review]
Thanks!

**Pull request merged**